common/src/View/Autosaver.cpp

```diff
@@ -49,13 +52,15 @@ namespace TrenchBroom {
             }
 
             const auto backupExtension = backupName.extension();
-            if (!StringUtils::isNumber(backupExtension)) {
+            if (!kdl::str_is_numeric(backupExtension)) {
                 return false;
             }
 
-            const auto no = StringUtils::stringToSize(backupName.extension());
-            return no > 0;
-
+            try {
+                return std::stoul(backupName.extension()) > 0u;
+            } catch (const std::exception&) {
```

Comment:
I don't think we should use these directly, also `catch (const std::exception&)` is dangerous.

Rather `nonstd::optional<size_t> kdl::str_to_size_t(const string_view& s)` or something.

Reply:
done